packages/view-types/scatterplot/src/ScatterplotOptions.js

```diff
@@ -121,124 +125,138 @@ export default function ScatterplotOptions(props) {
             onChange={handleTooltipsVisibilityChange}
             name="scatterplot-option-toltip-visibility"
             color="default"
+            inputProps={{
+              'aria-label': 'Checkbox for showing/hiding tooltips.',
+              id: 'scatterplot-set-tooltips-visible',
+            }}
           />
         </TableCell>
       </TableRow>
-      <TableRow>
-        <TableCell className={classes.labelCell}>
-          {observationsLabelNice} Set Label Size
-        </TableCell>
-        <TableCell className={classes.inputCell}>
-          <Slider
-            disabled={!cellSetLabelsVisible}
-            classes={{ root: classes.slider, valueLabel: classes.sliderValueLabel }}
-            value={cellSetLabelSize}
-            onChange={handleLabelSizeChange}
-            aria-labelledby="cell-set-label-size-slider"
-            valueLabelDisplay="auto"
-            step={1}
-            min={8}
-            max={36}
-          />
-        </TableCell>
-      </TableRow>
-      <TableRow>
-        <TableCell className={classes.labelCell}>
+      { cellSetLabelsVisible && (
```

Comment:
Is this change to using rendering vs. the `disabled` prop related to accessibility?

Reply:
This change has nothing to do with accessibility -- I did it for UX reasons. I thought it was confusing if the cell set labels are not visible, but the slider is still showing in a disabled mode. I thought it is confusing for the user to have a setting that is disabled. That said, you know your users better than I do, so if you disagree with this, I am happy to bring back the previous version of the code. Just let me know

Reply:
I read the comments further and I see that both of you are on the opinion that using `disabled` is better, so I reverted this change in commit e5371cbf